### PR TITLE
Guy/s152 connection

### PR DESCRIPTION
**Pull request by guyblink**
Fetching connections from new connection context

### CODE REVIEW

File: table/router.go [hunk 1]
```diff
@@ -69,12 +69,10 @@ func (p *CloudQuery) ExecuteAction(_ *plugin.ActionContext, request *plugin.Exec
 	if err != nil {
 		return nil, err
 	}
-	token := request.Parameters[utilities.AuthContextToken]
-	authid := request.Parameters[utilities.AuthContextCon]
-	ctx := context.WithValue(context.WithValue(
-		context.Background(), utilities.AuthContextToken, token),
-		utilities.AuthContextCon, authid)
-	res, err := tablePlugin.generate(ctx, qc)
+	conn, err := ctx.GetCredentials("aws")
```
Comment (danielsblinkops) [inline table/router.go:72]: handle error

File: table/router.go [hunk 2]
```diff
@@ -69,12 +69,10 @@ func (p *CloudQuery) ExecuteAction(_ *plugin.ActionContext, request *plugin.Exec
...
+
+	tableCtx := context.WithValue(context.Background(), utilities.AuthContextCon, conn)
```
Comment (danielsblinkops) [inline table/router.go:74]: Its no the same values, you have to setEnv (See what yuval did)

**Review by danielsblinkops — CHANGES_REQUESTED** [review]

File: table/router.go (hunk 2 above)
Comment (guyblink) [reply table/router.go:74]: Yeah, it just passes it in the context and sets it here: GetAwsConfig

File: table/router.go (hunk 1 above)
Comment (guyblink) [reply table/router.go:72]: Done

**Review by rimar-blink — APPROVED** [review]

**Pull request merged**